main.go

```diff
@@ -203,15 +218,16 @@ func getProcesses() []list.Item {
 	return processes
 }
 
-func killPort(pid string) {
-	pidInt, err := strconv.Atoi(pid)
-	if err != nil {
-		log.Error("Could not convert to process pid to int")
-	}
-	syscall.Kill(pidInt, syscall.SIGKILL)
-	if err != nil {
-		log.Error("Could not kill process")
+// returns a kill exec.Cmd for supported operating systems, otherwise an error.
+func killPortCmd(pid string) (*exec.Cmd, error) {
+	switch runtime.GOOS {
+	case "darwin":
+		return exec.Command("kill", pid), nil
+	case "linux":
+		return exec.Command("kill", pid), nil
```

Comment:
I did not test this branch as I'm on Mac. If you are on Linux, please test before deploying.